### PR TITLE
Add Therm 0.1.2

### PR DESCRIPTION
**Pull request by novia713**
initial commit, version Therm 0.1.2

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/therm.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'therm' do
+  version '0.1.2'
+  sha256 '4a1d43ef40f13d6cb6fadfa955489b7cd1f9976c4d332401565636ae543dbd1a'
+
+  url 'https://github.com/trufae/Therm/releases/download/0.1.2/Therm.app.zip'
```
Comment (commitay) [inline Casks/therm.rb]: Use `#{version}` in place of `0.1.2` and use double quotes.

File: Casks/therm.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  name 'Therm'
```
Comment (commitay) [inline Casks/therm.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md)

Example:
https://github.com/caskroom/homebrew-cask/blob/0480e1f119f2742586f35b65a4efbfba670757b8/Casks/autodmg.rb#L5-L8

File: Casks/therm.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+  homepage 'https://github.com/trufae/Therm'
+
+  app ''
```
Comment (commitay) [inline Casks/therm.rb]: `app 'Therm.app'`

**Review by commitay — DISMISSED** [review]

**Comment by novia713** [comment]
@commitay thank you for your attention & guidelines 
i hope it were now acceptable

**Review by commitay — APPROVED** [review]

**Review by victorpopkov — APPROVED** [review]

**Pull request merged**